src/vectors/vectorWindow.component.test.tsx

```diff
@@ -151,4 +157,54 @@ describe('Vector Window component', () => {
       },
     });
   });
+
+  it('checks that the vectors are using the default value of the slider (with the default vector upper bound)', async () => {
+    // Mock vector preferences
+
+    server.use(
+      http.get(`/users/preferences/${VECTOR_SKIP_PREFERENCE_NAME}`, () => {
+        return HttpResponse.json('2', { status: 200 });
+      }),
+      http.get(`/users/preferences/${VECTOR_LIMIT_PREFERENCE_NAME}`, () => {
+        return HttpResponse.json('5', { status: 200 });
+      })
+    );
+
+    const ref = React.createRef<WindowPortal>();
+    const mockResize = vi.fn();
+
+    Object.defineProperty(ref, 'current', {
+      value: {
+        getWindow: vi.fn(() => ({
+          Plotly: {
+            Plots: {
+              resize: mockResize,
+            },
+          },
+        })),
+      },
+      writable: true,
+    });
```

Comment:
Could this be mocked in the `beforeEach`? Saves the repetition of defining it twice.